agent/setup.go

```diff
@@ -119,6 +119,23 @@ func (a *Agent) buildDiscoveryConf(enabled module.Registry) discovery.Config {
 	}
 
 	for name := range enabled {
+		// TODO: properly handle module renaming
+		// We need to announce this change in Netdata v1.39.0 release notes and then remove this workaround.
+		// This is just a quick fix for wmi=>windows. We need to prefer user wmi.conf over windows.conf
+		// 2nd part of this fix is in /agent/job/discovery/file/parse.go parseStaticFormat()
+		if name == "windows" {
+			cfgName := "wmi.conf"
```

Comment:
Why cannot we check both files as temporary?

Reply:
What do you mean?

- we look for user wmi.conf
- if it doesn't exist, we look for windows.conf (both user and stock)

Reply:
My bad, I make a confusion about `nil` meaning.